source/_components/abode.markdown

```diff
@@ -42,21 +43,14 @@ There is currently support for the following device types within Home Assistant:
 
 ## Configuration
 
-To use Abode devices in your installation,
-add the following `abode` section to your `configuration.yaml` file:
+To use Abode devices in your installation, add your Abode account from the integrations page. Two factor authenticaion must be disabled on your Abode account. Alternatively, Abode can be configured by adding the following `abode` section to your `configuration.yaml` file:
 
 ```yaml
 # Example configuration.yaml entry
 abode:
   username: abode_username
   password: abode_password
-  name: Abode Alarm System
   polling: false
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

Reply:
Makes sense! Removed `polling: false` from the sample. I don't think another configuration sample is necessary since it's a feature that shouldn't really be used anyways.